app/scripts/modules/amazon/src/loadBalancer/configure/application/targetGroups.html

```diff
@@ -0,0 +1,89 @@
+        <div>
+          <div class="wizard-pod-row header">
+            <div class="wizard-pod-row-title">GROUP NAME</div>
+            <div class="wizard-pod-row-contents">
+              <input class="form-control input-sm"
+                          type="text"
+                          ng-model="targetGroup.name"
+                          required/>
+              <a href class="sm-label" ng-click="ctrl.removeTargetGroup($index)"><span class="glyphicon glyphicon-trash"></span></a>
+            </div>
+          </div>
+          <div class="wizard-pod-row">
+            <div class="wizard-pod-row-title">INCOMING CONNECTION</div>
+            <div class="wizard-pod-row-contents">
+              <span class="wizard-pod-content">
+                <label>Protocol</label> <select class="form-control input-sm inline-number" ng-model="targetGroup.protocol"
+                          ng-options="protocol for protocol in ['HTTP','HTTPS']"></select>
+              </span>
+              <span class="wizard-pod-content">
+                <label>Port</label> <input class="form-control input-sm inline-number" ng-model="targetGroup.port" type="text" required/>
+              </span>
+            </div>
+          </div>
+          <div class="wizard-pod-row">
+            <div class="wizard-pod-row-title">HEALTHCHECK</div>
+            <div class="wizard-pod-row-contents">
+              <span class="wizard-pod-content">
+              <label>Protocol</label> <select class="form-control input-sm inline-number" ng-model="targetGroup.healthCheckProtocol"
+                        ng-options="protocol for protocol in ['HTTP','HTTPS']"></select>
+              </span>
+              <span class="wizard-pod-content">
+                <label>Port</label> <input class="form-control input-sm inline-number" ng-model="targetGroup.healthCheckPort" type="text" required/>
+              </span>
+              <span class="wizard-pod-content">
+                <label>Path</label> <input class="form-control input-sm inline-text" ng-model="targetGroup.healthCheckPath" type="text" required/>
+              </span>
+              <span class="wizard-pod-content">
+                <label>Timeout</label> <input class="form-control input-sm inline-number" ng-model="targetGroup.healthTimeout" type="text" required/>
+              </span>
+              <span class="wizard-pod-content">
+                <label>Interval</label> <input class="form-control input-sm inline-number" ng-model="targetGroup.healthInterval" type="text" required/>
+              </span>
+            </div>
+          </div>
+          <div class="wizard-pod-row">
+            <div class="wizard-pod-row-title">HEALTHCHECK THRESHOLD</div>
+            <div class="wizard-pod-row-contents">
+              <span class="wizard-pod-content">
+                <label>Healthy</label> <input class="form-control input-sm inline-number" ng-model="targetGroup.healthyThreshold" type="text" required/>
```

Comment:
If we see a world where folks configure ALBs in pipelines, that makes sense; otherwise, it's maybe better to just make these number fields.

(obviously they are fine as text fields)